globe-view/index.ts

```diff
@@ -0,0 +1,155 @@
+import './style.css';
+import 'maplibre-gl/dist/maplibre-gl.css';
+import maplibregl from 'maplibre-gl';
+import {query} from '@deck.gl/carto';
+import {GeoJsonLayer} from '@deck.gl/layers'
+import {SimpleMeshLayer} from '@deck.gl/mesh-layers'
+import {SphereGeometry} from '@luma.gl/engine';
+import {
+  Deck,
+  COORDINATE_SYSTEM,
+  _GlobeView as GlobeView,
+  LightingEffect,
+  AmbientLight,
+  _SunLight as SunLight
+} from '@deck.gl/core';
+import AnimatedArcLayer from './AnimatedArcLayer';
+
+const apiBaseUrl = import.meta.env.VITE_API_BASE_URL;
+const accessToken = import.meta.env.VITE_API_ACCESS_TOKEN;
+const connectionName = 'carto_dw';
+const cartoConfig = {apiBaseUrl, accessToken, connectionName};
+
+const INITIAL_VIEW_STATE = {
+  longitude: 0,
+  latitude: 20,
+  zoom: 0
+};
+
+const TIME_WINDOW = 900; // 15 minutes
+const EARTH_RADIUS_METERS = 6.3e6;
+const MIN_TIME = 5;
+const MAX_TIME = 103289;
+const SEC_PER_DAY = 60 * 60 * 24;
+
+const ambientLight = new AmbientLight({
+  color: [255, 255, 255],
+  intensity: 0.5
+});
+
+const sunLight = new SunLight({
+  color: [255, 255, 255],
+  intensity: 2.0,
+  timestamp: 0
+});
+
+// create lighting effect with light sources
+const lightingEffect = new LightingEffect({ ambientLight, sunLight });
+
+// Look up the real date time from our artifical timestamp
+function getDate(t) {
+  const timestamp =
+    new Date("2020-01-14T00:00:00Z").getTime() + (t % SEC_PER_DAY) * 1000;
+  return new Date(timestamp);
+}
+
+const deck = new Deck({
+  canvas: 'deck-canvas',
+  initialViewState: INITIAL_VIEW_STATE,
+  controller: true,
+  views: new GlobeView(),
+  effects: [lightingEffect],
+  layers: []
+});
+
+// Add basemap
+const map = new maplibregl.Map({
+  container: 'map',
+  style:  {
+    version: 8,
+    sources: {},
+    layers: [
+      {
+        id: "background",
+        type: "background",
+        paint: { "background-color": "#000" }
+      }
+    ]
+  },
+  interactive: false
+});
+
+deck.setProps({
+  onViewStateChange: ({viewState}) => {
+    const {longitude, latitude, ...rest} = viewState;
+    map.jumpTo({center: [longitude, latitude], ...rest});
+  }
+});
+
+async function initialize() {
+  // Fetch Data from CARTO
+  const flightsSource = await query({
+    ...cartoConfig,
+    sqlQuery: `SELECT time1, time2, lat1, lat2, lon1, lon2, alt1, alt2 FROM cartobq.public_account.animated_deckgl_layer_flights`
+  });
+
+  const worldLandQuery = "SELECT the_geom FROM ne_50m_land_world";
```

Comment:
Use the [query](https://felixpalmer.github.io/deck.gl/docs/api-reference/carto/overview#support-for-other-deckgl-layers) function rather than making the request manually

Reply:
I think @menusal is doing a fetch because it's using a request to CARTO 2. I will move the data to carto 3 and you can use the query.

Reply:
In fact, we already have this table at `cartobq.public_account.ne_50m_land_world`

Reply:
Done